--- origsrc/libkgeomap-17.08.3/src/backends/CMakeLists.txt	2017-04-13 12:18:33.000000000 -0500
+++ src/libkgeomap-17.08.3/src/backends/CMakeLists.txt	2017-11-12 21:22:23.551025000 -0600
@@ -1,5 +1,5 @@
 
-add_definitions(-DKGEOMAP_MARBLE_ADD_LAYER)
+add_definitions(-DKGEOMAP_MARBLE_ADD_LAYER -DLIBKGEOMAP_STATIC_DEFINE)
 set(backend_map_marble_LIB_SRCS
     backendmarble.cpp
     backendmarblelayer.cpp
--- origsrc/libkgeomap-17.08.3/tests/CMakeLists.txt	2017-04-13 12:18:33.000000000 -0500
+++ src/libkgeomap-17.08.3/tests/CMakeLists.txt	2017-11-12 21:51:22.474583900 -0600
@@ -24,6 +24,7 @@ set(test_primitives_sources
 add_executable(kgeomap_test_primitives ${test_primitives_sources})
 target_link_libraries(kgeomap_test_primitives KF5KGeoMap Qt5::Core Qt5::Test)
 add_test(kgeomap_test_primitives ${EXECUTABLE_OUTPUT_PATH}/kgeomap_test_primitives)
+set_target_properties(kgeomap_test_primitives PROPERTIES COMPILE_DEFINITIONS LIBKGEOMAP_STATIC_DEFINE)
 
 # test the GeoCoordinates class
 
@@ -49,7 +50,7 @@ target_link_libraries(kgeomap_test_looku
 
 # test the marker model
 
-if(NOT WIN32)
+if(NOT WIN32 AND NOT CYGWIN)
     set(test_itemmarkertiler_sources
         test_itemmarkertiler.cpp
         ../src/kgeomap_common.cpp
